src/xdocs/cmdline.xml.vm

```diff
@@ -204,6 +204,20 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           &#xa0;&#xa0;&#xa0;&#xa0;MyClass.java
         </source>
       </div>
+      <p>
+        To run Checkstyle directly from the JAR file using
+        the <code>-jar</code> option when the JAR file and test file are not in same folder
+        or if CLI is launched from any other path. Download latest
```

Comment:
This crosses the line from "how to use checkstyle" (what we are responsible for) and "how to use the command line" (what the user is responsible for). This has no place in our docs.

Reply:
@nrmancuso , I thought https://checkstyle.org/cmdline.html was about "`how to use checkstyle via the command line`"
which is a merge of both "how to use checkstyle" and partly "how to use the command line".
I will remove it if you still stick by your decision but please reconsider because I don't see the purpose of having such well-framed docs if the user is still in doubt as to why the all jar isn't running

Reply:
Ok, I'll play devil's advocate then:

1. Why don't we tell them how to download it?
2. How do we "launch a CLI"?
3. Is `~` always the user's home directory in all OS's/terminals/shells?

Hopefully, you can see where this is going. Also, more documentation isn't necessarily better (and in my experience, it usually isn't). Now, new users have more "stuff" to wade through. 

What we have here is better fodder for a Baeldung article or blog post.

Reply:
>Sorry, crossing your suggestion was not my intention.

No need for sorry, this is is how things get discussed and we find mutually acceptable solutions (or realize that we never had a problem in the first place) :)

I do not see value in this update, personally. Like I said above, I think this crosses the line of what the user is responsible for.

What may be more beneficial for new users is a blog post from you :) It seems like you enjoy writing.

Reply:
> What may be more beneficial for new users is a blog post from you :) It seems like you enjoy writing.

I would say "working" in place of "writing" :)
A blog can be arranged if you really think it can be tagged to our docs @nrmancuso